packages/orbit-components/src/Card/index.tsx

```diff
@@ -48,46 +27,34 @@ const Card = ({
   header,
   spaceAfter,
   dataA11ySection,
-}: Props) => {
-  const [expandedSections, setExpandedSections] = React.useState<number[]>([]);
-  const theme = useTheme();
-
-  // handles array of expanded sections
-  const addSection = React.useCallback((index: number) => {
-    setExpandedSections(prev => (prev.indexOf(index) === -1 ? [...prev, index] : prev));
-  }, []);
-
-  const removeSection = React.useCallback((index: number) => {
-    setExpandedSections(prev =>
-      prev.indexOf(index) !== -1 ? prev.filter(val => val !== index) : prev,
-    );
-  }, []);
-
-  // Currently disable that code, becuase of IE 11, where it does not work
-  // It will be fixed later, when we'll find solution
-  const renderSection = item => {
-    if (React.isValidElement(item)) {
-      // if (item.props.children && item.type.name !== "CardSection") {
-      //   return React.createElement(CardSection, {
-      //     ...item.props.children.props,
-      //     key: index,
-      //   });
-
-      return React.cloneElement(item);
+}: Props) {
+  const marginStyles = (() => {
+    if (margin == null) {
+      return {};
     }
-
-    return null;
-  };
+    if (typeof margin === "string" || typeof margin === "number") {
+      return { margin };
+    }
+    return {
+      marginTop: margin.top,
+      marginRight: margin.right,
+      marginBottom: margin.bottom,
+      marginLeft: margin.left,
+    };
+  })();
 
   return (
-    <StyledCard
-      // TODO: remove spaceAfter in the next major version
-      margin={spaceAfter ? { bottom: getSpacingToken({ spaceAfter, theme }) } : margin}
-      data-test={dataTest}
+    <div
       id={id}
+      className={cx(
+        "[&>*]:border-elevation-flat-border-color [&>*:first-child]:rounded-t-normal [&>*:last-child]:rounded-b-normal [&>*:first-child]:border-t-[1px]",
```

Comment:
I assume this dev replaces the `StyledCard` but I don't recognize these styles there. I see a different set of styles, not matched here
```
    width: 100%;
    box-sizing: border-box;
    position: relative;
    font-family: ${theme.orbit.fontFamily};
```

Reply:
added font-family. other styles are not needed